src/styles/blocks/page.scss

```
@@ -0,0 +1,38 @@
+.page {
+  font-family: Inter, sans-serif;
+  color: $c-general;
+  scroll-behavior: smooth;
```

```suggestion
  scroll-behavior: smooth;
  scroll-padding-top: 40px;
```

Nice prop to have for unstick anchored section from the top of the screen after the scroll.

---

nice! thanks